### ISSUE TITLE
optional `spec.imageChartStorage.s3.certificateRef` validation failed

### ISSUE THREAD

**Issue by bitsf** [goharbor/harbor-operator#450]
reporting validation error of spec.imageChartStorage.s3.certificateRef

**Comment by bitsf** [comment]
add `spec.inClusterStorage.minIOSpec.redirect.expose.tls.certificateRef` seems can workaround

**Comment by bitsf** [comment]
also certificateRef seems lost `omitempty`

**Comment by steven-zou** [comment]
```
2021-03-04T03:23:30.493Z        ERROR   controller      Reconciler error        {"reconcilerGroup": "goharbor.io", "reconcilerKind": "HarborCluster", "controller": "harborcluster", "name": "harborcluster-sample", "namespace": "cluster-sample-ns", "error": "reconcile harbor service error: Harbor.goharbor.io \"harborcluster-sample-harbor\" is invalid: spec.imageChartStorage.s3.certificateRef: Invalid value: \"\": spec.imageChartStorage.s3.certificateRef in body should match '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'"}
```

**Comment by steven-zou** [comment]
this issue has been fixed by #479
